### DISCUSSION TITLE
Laser welding with Baldr/MALAMUTE

### DISCUSSION THREAD

**Discussion by am-tc01** [idaholab/moose#23032] [Q&A General] [answered]
Hi,

I need to simulate laser welding and melt pool formation with fluid flow and surface deformation like the example given on Moose homepage. Is it possible to employ ALE using the current moose version to do this? 
Following a discussion [here](https://github.com/idaholab/moose/discussions/20341#discussioncomment-2205786), I understood that this example was simulated in Baldr or Malamute. Is is possible to get access to that?

Regards.

**Comment by GiudGiud** [comment]
@jiangwen84 will know more but I think the situation has not changed since this other post.

**Reply by am-tc01** [reply]
Thanks @GiudGiud. I actually applied for an INL HPC account stating the reason that I need access to MALAMUTE, but my application was denied. I was told that NCRC doesn't manage MALAMUTE but it is the Technology Department (also cc'd in that email) that manages this and can advise on how to obtain the code. I then contacted the Technology Department but haven't heard anything from them since. 
Do you have any more info @jiangwen84 or @cticenhour ?
Thanks!

**Reply by cticenhour** [reply]
Getting the INL HPC account through the NCRC is still the way to get started, as MALAMUTE access is not possible without an account. INL is closed for the holiday season, which may be why you haven't gotten a reply from Technology Deployment. I can check in on this in January, once INL is back to work.

**Reply by am-tc01** [reply]
Thanks @cticenhour. My application for an account on INL HPC was denied because of the reason I stated above. But I can wait for an answer from Technology Deployment, and if then needed I'll re-apply for the HPC account.
Happy holidays!

**Reply by GiudGiud** [reply]
Any update on this @am-tc01 ?

**Reply by am-tc01** [reply]
@GiudGiud No, I haven't heard anything from Technology Deployment. 
@cticenhour I wonder if you had time to check in on this?
Thanks!

**Reply by cticenhour** [reply]
Did you send another email to them after the holidays? Your old email may have gotten left behind in their inbox, so it's worth giving them a reminder that you're waiting. Also, if I need to follow up with them directly, I would need more information from you - like, your current university/institution and name.

**Reply by am-tc01** [reply]
Hi @cticenhour 

I have now sent an email to Technology Deployment and cc'ed you as well.
Hope to hear back from them soon.
Thanks!

**Reply by am-tc01** [reply]
Hi, Just wondering if there is any update on this @cticenhour?

**Reply by cticenhour** [reply]
Spoke to the manager over the NCRC, and am working on resurrecting your case on my end. Will let you know when I know more.

**Reply by cticenhour** [reply]
Hi again - you should have received an email a bit ago regarding MALAMUTE access from the NCRC. I am CC-ed on that email message, so please feel free to reach out directly if you have other issues. Thanks again for your interest in MALAMUTE!

**Comment by cticenhour** [comment, marked as answer]
@am-tc01 Good news - MALAMUTE is now open source and can be found at https://github.com/idaholab/malamute. 

Given this new status for MALAMUTE, do you still require INL HPC access? Please advise. Thanks!

**Reply by am-tc01** [reply]
Hi @cticenhour Thanks for the info. Yes that is indeed a good news!
No, I wouldn't require INL HPC access anymore. I'll reply to their email as well.

On a separate note, I have already started looking into the MALAMUTE code and tried to run some of the examples. Some of them worked, but some didn't converge and few gave errors. But perhaps I can create separate tickets for those in the Malamute discussion forum.

**Reply by cticenhour** [reply]
Yes, please open a discussion on the MALAMUTE repository for any examples that aren't running as expected.
